Tech docs (#105)
microsoft/ReactorPlanning: docs/README.md

## Software

The key piece of software you'll need if streaming directly is [OBS Studio](https://obsproject.com/). OBS Studio (or simply OBS) is a powerful OSS streaming suite, and gives you complete control over building the scene for your viewers. Admittedly, OBS Studio can be a little intimidating for anyone who hasn't worked with video editing software in the past.

- [Installing and configuring OBS for streaming](./obs-install.md)
- [Installing and configuring the Reactor scene collection](./obs-scenes.md) (in progress)

### Software tl;dr

- Actually, on this one you're going to need to read everything :-)

## Hardware guide

While no special hardware is required, a couple of pieces can help elevate your game. We've put together a [hardware guide](./hardware.md) to help you get started.

### Hardware tl;dr

- Have a good mic
  - The one in your camera/laptop isn't going to be high quality
  - Chances are the one in your headset isn't either
- Consider an upgraded camera
- Don't be back-lit

## Preparing your session

> The final steps to preparing to stream is to set the scene. This includes both your physical space and tone and timber. We have a set of [streaming guidelines](digital-stream-guidance.md) to help you hit the right notes.

Reviewer: Is this supposed to link to the delivering-your-stream.md doc instead? Looks like the .md referenced got deleted.

### Session prep tl;dr

- Take a minute to look at your backdrop for streaming
- 100-200 level content; don't assume people meet pre-requisites
- Ensure you have a mentor to help answer questions
- Tweet out links to your session (or whatever other social media you may use)

## Streaming

### Streaming tl;dr

- Start the stream about 5 minutes beforehand with the "starting soon" scene
  - Make sure your mic is turned **off**!
- Start your presentation about 1 minute after the scheduled time

>   - Make sure your mice is turned **on**!

Reviewer: ```suggestion
  - Make sure your mic is turned **on**!
```

- Look at the camera!
- Maintain a light, conversational tone
- Repeat all questions before you answer them
  - Bonus points for saying the name of the person who asked

> - DON'T PANIC!!

Reviewer: <3 Great tip!

# Streaming guide

> Want to stream but don't know where to start? We've got you covered! This guide is designed to help get you started with streaming. The only assumed knowledge is experience with conference calls and sharing your desktop/app on an app like Teams, Skype or Slack. Please note, you will likely need to spend a couple of hours getting a base setup put together.

Reviewer 2: Are we not an oxford comma team?
```suggestion
Want to stream but don't know where to start? We've got you covered! This guide is designed to help get you started with streaming. The only assumed knowledge is experience with conference calls and sharing your desktop/app on an app like Teams, Skype, or Slack. Please note, you will likely need to spend a couple of hours getting a base setup put together.
```

## Software

> The key piece of software you'll need if streaming directly is [OBS Studio](https://obsproject.com/). OBS Studio (or simply OBS) is a powerful OSS streaming suite, and gives you complete control over building the scene for your viewers. Admittedly, OBS Studio can be a little intimidating for anyone who hasn't worked with video editing software in the past.

Reviewer 2: ```suggestion
The key piece of software you'll need if streaming directly is [OBS Studio](https://obsproject.com/). OBS Studio (or simply OBS) is a powerful, open source streaming suite, and gives you complete control over building the scene for your viewers. Admittedly, OBS Studio can be a little intimidating for anyone who hasn't worked with video editing software in the past.
```

- [Installing and configuring OBS for streaming](./obs-install.md)
- [Installing and configuring the Reactor scene collection](./obs-scenes.md) (in progress)

### Software tl;dr

> - Actually, on this one you're going to need to read everything :-)

Reviewer 2: 😆

## Hardware guide

While no special hardware is required, a couple of pieces can help elevate your game. We've put together a [hardware guide](./hardware.md) to help you get started.

### Hardware tl;dr

- Have a good mic
  - The one in your camera/laptop isn't going to be high quality

>   - Chances are the one in your headset isn't either

Reviewer 2: I'd probably describe this a bit more. I would say something like:
```suggestion
  - Chances are the one in your headset is ok enough if you're only planning on streaming once, but not high quality
```

- Consider an upgraded camera
- Don't be back-lit

## Preparing for your session

> The final steps to preparing to stream is to set the scene. This includes both your physical space and tone and timber. Here's a few tips to help [prepare your stream](./peparing-your-session.md).

Reviewer 2: Sentence felt strange. what do you mean by tone and timber?

```suggestion
The final steps to preparing to stream is to set the scene. This includes both your physical space and your digital desktop and tools. Here's a few tips to help [prepare your stream](./peparing-your-session.md).
```

> The final steps to preparing to stream is to set the scene. This includes both your physical space and tone and timber. Here's a few tips to help [prepare your stream](./peparing-your-session.md).

Author: ```suggestion
The final steps to preparing to stream is to set the scene. This includes both your virtual and physical space. Here's a few tips to help [prepare your stream](./peparing-your-session.md).
```

### Session prep tl;dr

- Take a minute to look at your backdrop for streaming
- Double check your lighting!

> - Practice talking into the void

Reviewer 2: ```suggestion
- Practice talking into the void while looking at a camera and not your monitor
```

- 100-200 level content; don't assume people meet pre-requisites

> - Ensure you have a moderator to help answer questions

Reviewer 2: ```suggestion
- Ensure you have a moderator to help answer questions
- Equip your moderator with links that will likely be useful to share in the chat
```

- Have great demos!
- Tweet out links to your session (or whatever other social media you may use)

## Streaming

Now that you've got everything done, you're ready to go live! We've put together a bit of [help for the day of streaming](./delivering-your-session.md), including the overall run of show.

### Streaming tl;dr

- Start the stream about 5 minutes beforehand with the "starting soon" scene

>   - Make sure your mic is turned **off**!

Reviewer 2: ```suggestion
  - Make sure your mic is turned **off** in OBS!
```

- Start your presentation about 1 minute after the scheduled time

>   - Make sure your mic is turned **on**!

Reviewer 2: ```suggestion
  - Make sure your mic is turned **on** in OBS!
```

- Look at the camera!
- Maintain a light, conversational tone
- Repeat all questions before you answer them

>   - Bonus points for saying the name of the person who asked

Reviewer 2: ```suggestion
  - Bonus points for saying the name of the person who asked
- Leave time to answer questions at the end
```
